packages/web-components/src/message-bar/message-bar.ts

```diff
@@ -0,0 +1,70 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+import { MessageBarIntent, MessageBarLayout, MessageBarPoliteness, MessageBarShape } from './message-bar.options.js';
+
+/**
+ * A Message Bar Custom HTML Element.
+ *
+ * @slot actions - Content that can be provided for the actions
+ * @slot close - Content that can be provided for the close button
+ * @slot - The default slot for the content
+ * @public
+ */
+export class MessageBar extends FASTElement {
+  /**
+   * Sets the aria-labelledby attribute of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `aria-labelledby`
+   */
+  @attr({ attribute: 'aria-labelledby' })
+  ariaLabelledBy: string | null = null;
+
+  /**
+   * Sets the shape of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `shape`
+   */
+  @attr
+  shape: MessageBarShape = 'rounded';
+
+  /**
+   * Sets the layout of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `layout`
+   */
+  @attr
+  layout: MessageBarLayout = 'singleline';
+
+  /**
+   * Sets the intent of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `intent`
+   */
+  @attr
+  intent: MessageBarIntent = 'info';
+
+  /**
+   * Sets the politeness of the control.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `politeness`
+   */
+  @attr
+  politeness: MessageBarPoliteness = 'polite';
```

Comment:
Status is a role which actually _is_ a live region defaulting the politeness. My bent here is to use documentation and/or an example to illustrate how you would override that rather than including a separate attribute. If I wanted to override the aria-live attribute, I think the easiest method would be:

```html
<message-bar aria-live="assertive"></message-bar>
```

Reply:
Got it. I removed the `politeness` attribute and added a Assertive example to the Storybook.